public/app/features/explore/Logs/Logs.test.tsx

```diff
@@ -95,41 +97,21 @@ describe('Logs', () => {
     });
   });
 
-  const getComponent = (partialProps?: Partial<ComponentProps<typeof Logs>>, logs?: LogRowModel[]) => {
+  const getComponent = (
+    partialProps?: Partial<ComponentProps<typeof Logs>>,
+    dataFrame?: DataFrame,
+    logs?: LogRowModel[]
+  ) => {
     const rows = [
       makeLog({ uid: '1', rowId: 'id1', timeEpochMs: 1 }),
       makeLog({ uid: '2', rowId: 'id2', timeEpochMs: 2 }),
       makeLog({ uid: '3', rowId: 'id3', timeEpochMs: 3 }),
     ];
 
-    const testDataFrame = {
-      fields: [
-        {
-          config: {},
-          name: 'Time',
-          type: FieldType.time,
-          values: ['2019-01-01 10:00:00', '2019-01-01 11:00:00', '2019-01-01 12:00:00'],
-        },
-        {
-          config: {},
-          name: 'line',
-          type: FieldType.string,
-          values: ['log message 1', 'log message 2', 'log message 3'],
-        },
-        {
-          config: {},
-          name: 'labels',
-          type: FieldType.other,
-          typeInfo: {
-            frame: 'json.RawMessage',
-          },
-          values: ['{"foo":"bar"}', '{"foo":"bar"}', '{"foo":"bar"}'],
-        },
-      ],
-      length: 3,
-    };
+    const testDataFrame = dataFrame ?? getMockLokiFrame();
     return (
       <Logs
+        datasourceType={'loki'}
```

Comment:
This should not be needed anymore.

Reply:
It is also at other places in test file.

Reply:
Fixed!